crypto/obj/obj_dat.h

```diff
@@ -8757,6 +8757,11 @@ static const ASN1_OBJECT kObjects[NUM_NID] = {
     {"AuthANY", "auth-any", NID_auth_any, 0, NULL, 0},
     {"CECPQ2", "CECPQ2", NID_CECPQ2, 0, NULL, 0},
     {"CECPQ2b", "CECPQ2b", NID_CECPQ2b, 0, NULL, 0},
+    {NULL, NULL, NID_undef, 0, NULL, 0},
```

Comment:
What are the NULL ones for?

Reply:
Good catch, it turns out `CECPQ2b` was assigned OID 960 and `oqs_kemdefault` was assigned OID 963 (as seen [here](https://github.com/open-quantum-safe/boringssl/pull/1/files#diff-c605c4f6d6747678deeafeb41eaf91f7)). The 2 `NID_undef`s were being printed to bridge this gap.